S/Spasm/build_tarballs.jl

```diff
@@ -0,0 +1,60 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Spasm"
+version = v"1.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/laurentbartholdi/spasm.git", "072719a40c837e447dfe4ae9e4941c60d9a28eda"),
+    GitSource("https://github.com/linbox-team/givaro.git", "fc6cac7820539c900dde332326c71461ba7b910b"),
+    GitSource("https://github.com/linbox-team/fflas-ffpack.git", "94aa88263f5c6032adb5c86bf806f007fec7aded"),
```

Comment:
Why these aren't separate packages?

Reply:
It would be difficult to do that cleanly, because fflas-ffpack is a pure C++ package while Spasm is pure C. However, I could pull out Givaro as C++ package to have it compiled separately. I don't see the benefit.

Reply:
> It would be difficult to do that cleanly, because fflas-ffpack is a pure C++ package while Spasm is pure C.

I don't see how that makes anything difficult.

> However, I could pull out Givaro as C++ package to have it compiled separately. I don't see the benefit.

I do: modularity.